graveler/sstable/manager.go

```diff
@@ -16,10 +16,10 @@ type Manager struct {
 	fs    pyramid.FS
 	// TODO(ariels): Replace with loggers constructed from context.
 	logger logging.Logger
-	hash   hash.Hash
+	hash   crypto.Hash
```

Comment:
Amusingly, neither of `{hash,crypto}.Hash` is the other: `crypto.Hash` might better be called `crypto.HashAlgorithm`, but that may be 10 years too late. ;-(

Reply:
guess they like it simple - I liked the fact that my `int` can create an hash instance